hydro_std/Cargo.toml

```diff
@@ -17,9 +17,9 @@ all-features = true
 [dependencies]
 hydro_lang = { path = "../hydro_lang", version = "^0.13.0" }
 stageleft = "0.8.1"
-serde = "1"
 hdrhistogram = "7.5.4"
-stats-ci = "0.1.1"
+stats-ci = { git = "https://github.com/hydro-project/stats-ci.git", branch = "main", features = ["serde"] }
```

Comment:
This is going to cause trouble because we can't have Git dependencies in things published to crates.io. We should either publish this fork, or just implement the relevant logic ourselves. @MingweiSamuel thoughts?

Reply:
Or upstream the changes and have https://github.com/xdefago/stats-ci publish it (eventually)